packages/server/src/config.ts

```diff
@@ -17,20 +17,15 @@ export const getServerConfig = () =>
     ProsopoServerConfigSchema.parse({
         defaultEnvironment: process.env.PROSOPO_DEFAULT_ENVIRONMENT, // enviromental variables
         defaultNetwork: process.env.PROSOPO_DEFAULT_NETWORK,
-        serverUrl: getServerUrl(),
+        serverUrl:
```

Comment:
This links to a closed issue - why are we changing this? I don't understand the comment

Reply:
I didn't change this
Should I change this to origin?